src/components/ui/avatar.tsx

```diff
@@ -0,0 +1,50 @@
+'use client';
+
+import * as React from 'react';
+import * as AvatarPrimitive from '@radix-ui/react-avatar';
```

Comment:
Thank you for your great work!

This is using wildcard imports. 
Would you modify it according to the Style Guide?

Reply:
@HrRn Hello!
Thank you for mentioning that, and I have indeed fixed this problem,